client/my-sites/domains/domain-search/index.tsx

```diff
@@ -435,20 +469,23 @@ export default connect(
 			selectedSiteId: siteId,
 			selectedSiteSlug: getSelectedSiteSlug( state ),
 			domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY ),
-			isSiteUpgradeable: isSiteUpgradeable( state, siteId ),
-			isSiteOnMonthlyPlan: isSiteOnMonthlyPlan( state, siteId ),
+			isSiteUpgradeable: !! siteId && isSiteUpgradeable( state, siteId ),
```

Comment:
Just thought out loud - I don't like this extra condition, but unfortunately it's indeed the best way, to avoid rewriting all functions like `isSiteUpgradeable` :(